phpunit/abstracts/CommonITILRecurrentTest.php

```diff
@@ -505,9 +511,7 @@ public function testComputeNextCreationDate()
             $current_date = $row['current_date'] ?? null;
 
             // Handle dynamic date
-            $date_to_restore = null;
```

Comment:
I think session is already reset between tests now, no need for specific code here.